### ISSUE TITLE
Enable filtering for long lists in CV fields (check one use case)

### ISSUE THREAD

**Issue by poikilotherm** [IQSS/dataverse#7889]
- This is a follow up for #6000, #6339 and relates to #7844 / #7877
- Tagging a few important people here, as UI related: @mheppler @TaniaSchlatter @scolapasta @djbrooke 

Can we please enable conditionally a filter input on controlled vocabulary fields with long lists? There is a nice example for this easy to use `filter` tag for [`<p:selectOneMenu>` Advanced Field](http://www.primefaces.org:8080/showcase/ui/input/oneMenu.xhtml)

We could:
1. Enable filtering for longish lists only (simple EL expression on the length of the list)
2. Make the length configurable (MPCONFIG for the win), defaulting to maybe 20? 30?

This should be a rather small code change, yet it will improve a situation open since #6000. I know @TaniaSchlatter wanted a more "categorized search" for really long lists, but maybe we can just make a start?

I know many of the UI people are eager to see some visual input before taking a final decision. Maybe you could give me a hint if this is at least considerable, so I can create a proof of concept in a PR. (So we don't waste dev time on things that don't stand a chance)

**Comment by cmbz** [comment]
@poikilotherm is this feature still needed/desired? If so, we will add it for consideration in the SPA.

**Comment by scolapasta** [comment]
@poikilotherm we just checked and we do see that it does have it in the jsf code (e.g. identifier type). So I'll go aheand and clse this and you could repoen if there's anything we're missing.
